akd/src/directory.rs

```diff
@@ -486,6 +546,12 @@ pub async fn get_key_history_hashes<S: Storage + Sync + Send, H: Hasher>(
     Ok((root_hashes, previous_root_hashes))
 }
 
+fn vec_to_u8_arr(vector_u8: Vec<u8>) -> [u8; 32] {
```

Comment:
Definitely add a comment here that this TRUNCATES, so if anyone uses this in the future they know the behavior

Reply:
Makes sense and commented in both places this is used.